src/lib.rs

```diff
@@ -101,6 +112,13 @@ enum Alg {
     EdDsa = -8,
     Totp = -9, // Unassigned, we use it for TOTP
 
+    #[cfg(feature = "mldsa44")]
+    Mldsa44 = -87,
+    #[cfg(feature = "mldsa65")]
+    Mldsa65 = -88,
+    #[cfg(feature = "mldsa87")]
+    Mldsa87 = -89,
```

Comment:
Can you please add a comment that mentions how these are chosen?  AFAIS they are not standardized yet, right?